test/change.test.js

```diff
@@ -104,6 +134,29 @@ describe('Change', function() {
       });
     });
 
+    describe('when a change doesnt exist - promise variant', function() {
+      beforeEach(function(done) {
+        var test = this;
+        Change.findOrCreateChange(this.modelName, this.modelId)
+        .then(function(result) {
+          test.result = result;
+          done();
+        })
+        .catch(function(err) {
+          return done(err);
+        });
```

Comment:
Please simplify - pass `done` as the second arg of `then`, remove the `catch` clause.